nat-lab/tests/utils/command_grepper.py

```diff
@@ -88,15 +90,14 @@ async def _run_check(
     ) -> bool:
         while True:
             process = await self._connection.create_process(self._check_cmd).execute()
+            self._last_stdout = process.get_stdout()
```

Comment:
Might be irrelevant but question - does it return full or partial stdout via `get_stdout()`? If it's full, then the `last` part is no necessary :)

Reply:
It does return full, just named it last, because it is, well, the last one received :smile: